kyaml/pathutil/pathutil_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package pathutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSubDirsWithFile(t *testing.T) {
+	res, err := SubDirsWithFile("test/data/dataset1", "Krmfile")
+	if !assert.NoError(t, err) {
+		t.FailNow()
+	}
+	expected := []string{"test/data/dataset1/Krmfile",
```

Comment:
If these are just empty files, it seems like it might be easier to just create the structure with code for the test instead of having the files checked in.

Reply:
Done.